src/olympia/addons/tests/test_serializers.py

```diff
@@ -1233,14 +1262,16 @@ def test_basic(self):
         result = self.serialize()
         assert (
             set(result.keys()) ==
-            {'id', 'name', 'icon_url', 'is_recommended', 'type', 'url'}
+            {'id', 'name', 'icon_url', 'is_recommended', 'type', 'url',
+             'promoted'}
         )
         assert result['id'] == self.addon.pk
         assert result['name'] == {'en-US': str(self.addon.name)}
         assert result['icon_url'] == absolutify(self.addon.get_icon_url(64))
         assert result['is_recommended'] == self.addon.is_recommended is False
         assert result['type'] == 'extension'
         assert result['url'] == self.addon.get_absolute_url()
+        assert result['promoted'] == self.addon.promoted
```

Comment:
can we add on the assertion that self.addon.promoted is `None`  (that's the only time that `self.addon.promoted` should equal the serialized JSON)